app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from errors import (NotVaccinatedError,
+                    OutdatedVaccineError,
+                    NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(expiration_date)
```

Comment:
Ensure that the `OutdatedVaccineError` is raised with a descriptive message. The current implementation correctly passes the expiration date, which is good for debugging and understanding the error context.